### ISSUE TITLE
Reason for mapping application.yaml to grpc

### ISSUE THREAD

**Issue by ksemaev** [hashgraph/hedera-mirror-node#898]
Hi guys. You have this line in compose https://github.com/hashgraph/hedera-mirror-node/blob/master/docker-compose.yml#L26

But this file https://github.com/hashgraph/hedera-mirror-node/blob/master/application.yml  has options only for the importer according to it's structure https://github.com/hashgraph/hedera-mirror-node/blob/master/application.yml#L1-L3 , while grpc has no such options. Should it even be mapped there?

**Comment by steven-sheehy** [comment]
Technically the application.yaml in the root doesn't have any importer properties either as they are all commented out. The mapping is there for both importer and grpc to provide users an example of how to pass configuration to each of the components. We used not have such a mapping and just documented how to enable passing custom config and people complained it was too hard so we landed with this approach.
